extensions/amp-social-share/0.1/amp-social-share-config.js

```diff
@@ -156,4 +156,16 @@ const BUILTINS = {
       },
     },
   },
+  'whatsapp': {
+    'url': 'whatsapp://send',
+    'text': '\u00A0', // Use a nbsp to ensure the anchor isn't collapsed
+    'params': {
```

Comment:
This actually changed a bit since this PR. Mind rebasing with master and I'll redo the review?